### ISSUE TITLE
CanvasController loads DEFAULT theme on initial load

### ISSUE THREAD

**Issue by n8rzz** [openscope/openscope#696]
Discovered as part of #670, the `CanvasController` will _always_ instantiate with the `DEFAULT` theme.  There is an available `_setTheme()` method, but this is only called as a result of changing the selected option in the options menu.

**Comment by erikquinn** [comment]
Also, it defaults hard-codedly to `THEME.DEFAULT` and should instead be passed through the constructor chain so there is a single source of truth for which theme to start a session with.

**Comment by kmandagie** [comment]
@sam-irl  are you already work in this issue?

**Comment by n8rzz** [comment]
@kmandagie I don't believe so.  I would hold off on starting anything on this until (at least) #726 (#767) is done.  there is a potential for heavy conflicts there

**Comment by n8rzz** [comment]
blocked by #726
